packages/block-ui/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@availity/block-ui",
+  "version": "0.1.0",
```

Comment:
If we are going to change the scope of the package, let's make this a `1.0.0` release, otherwise `BREAKING CHANGE:` commits will bump minor versions. I'm 50/50 on that change though. We can move the code here to make it easier to maintain/deploy without changing the name.

If we change, definitely need to add a note and run `npm deprecate` like was mentioned. We should also provide a codemod so that internal teams and external users can upgrade easily. 

If we don't change, should update the version to match current `react-block-ui` version